test/local_example_tests.robot

```diff
@@ -0,0 +1,67 @@
+*** Settings ***
+Documentation           Tests to verify the csolution examples can be compiled and executed

```

Comment:
Not sure if this could be done, I need to investigate this. Otherwise, I shall rephrase this string to fit into the permissible limits keeping the meaning same

Reply:
I belive the simplest way is to add a CLI argument when invoking robot:
```sh
robot --consolewidth=150
```
(The default value is 78 according to the help.)